synchronization/src/cyface/java/de/cyface/synchronization/CyfaceSyncAdapter.java

```diff
@@ -167,28 +170,28 @@ public void onPerformSync(final @NonNull Account account, Bundle extras, String
                     Cursor directionsCursor = null;
                     try {
```

Comment:
Is it possible to reduce the code duplication inside this try statement?

Reply:
Habe den Großteil der onPerformSync Methode zumindest in 2 Untermethoden geschoben, in dem Fall hier ist mir aber spontan nichts eingefallen, wie ich das reduziere. Wird aber ja nicht das letzte Mal sein, dass wir hier refactorn, hat eigtl. nichts mit dem Task des PR zu tun.